<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblDivider2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>19</y>
      <width>20</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(176, 214, 21);
border: 1px solid rgb(192, 191, 188);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblBorder1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>20</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(176, 214, 21);
border: 1px solid rgb(192, 191, 188);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpAEDView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>451</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font-weight: bold;
background-color: rgb(119, 118, 123);</string>
    </property>
    <property name="title">
     <string>AED View</string>
    </property>
    <widget class="QLabel" name="lblStep3">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>50</y>
       <width>91</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 4px solid  rgb(175, 193, 204);
border-radius: 6px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Downloads/Pics/Step3.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="lblStep6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>370</y>
       <width>91</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 4px solid  rgb(175, 193, 204);
border-radius: 6px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Downloads/Pics/Step6.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="lblStep2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>91</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 4px solid  rgb(175, 193, 204);
border-radius: 6px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Downloads/Pics/Step2.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="lblStep5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>450</y>
       <width>91</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 4px solid  rgb(175, 193, 204);
border-radius: 6px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Downloads/Pics/Step5.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="btnShockIndicator">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>380</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);
font-size: 45px;
font-weight: bold;
background-color: rgb(133, 172, 190);
border: 4px solid rgb(175, 193, 204);
border-radius: 25px;</string>
     </property>
     <property name="text">
      <string>♡</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblStep4">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>370</y>
       <width>91</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 4px solid  rgb(175, 193, 204);
border-radius: 6px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Downloads/Pics/Step4.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="lblStep1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>91</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 4px solid  rgb(175, 193, 204);
border-radius: 6px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Downloads/Pics/Step1.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="lblPath1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>20</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(176, 214, 21);
border: 1px solid rgb(192, 191, 188);
font-size: 14px;
font-weight: bold;
</string>
     </property>
     <property name="text">
      <string>↑</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lblPath2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(176, 214, 21);
border: 1px solid rgb(192, 191, 188);
font-size: 14px;
font-weight: bold;

</string>
     </property>
     <property name="text">
      <string>→</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblPath3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(176, 214, 21);
border: 1px solid rgb(192, 191, 188);
font-size: 14px;
font-weight: bold;

</string>
     </property>
     <property name="text">
      <string>→</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblPath4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>130</y>
       <width>20</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(176, 214, 21);
border: 1px solid rgb(192, 191, 188);
font-size: 14px;
font-weight: bold;
</string>
     </property>
     <property name="text">
      <string>↓</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblPath5">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>450</y>
       <width>20</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(176, 214, 21);
border: 1px solid rgb(192, 191, 188);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblPath6">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>480</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(176, 214, 21);
border: 1px solid rgb(192, 191, 188);
font-size: 14px;
font-weight: bold;
</string>
     </property>
     <property name="text">
      <string>←</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblDivider1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>570</y>
       <width>451</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(176, 214, 21);
border: 1px solid rgb(192, 191, 188);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPowerButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>600</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(175, 193, 204);
font-size: 45px;
font-weight: bold;
background-color: rgb(133, 172, 190);
border: 4px solid rgb(175, 193, 204);
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string>O</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>600</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(175, 193, 204);
font-size: 25px;
font-weight: bold;
background-color: rgb(133, 172, 190);
border: 4px solid rgb(175, 193, 204);
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string>STATUS</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblAdult">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>520</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nimbus Mono L</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold; font-size: 18px;</string>
     </property>
     <property name="text">
      <string>ADULT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblChild">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>540</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nimbus Mono L</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
font-size: 18px</string>
     </property>
     <property name="text">
      <string>CHILD</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpAEDView_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>270</y>
      <width>301</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font-weight: bold;
background-color: rgb(119, 118, 123);</string>
    </property>
    <property name="title">
     <string>Action Log</string>
    </property>
    <widget class="QPushButton" name="btnPowerButton_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>580</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(175, 193, 204);
font-size: 45px;
font-weight: bold;
background-color: rgb(133, 172, 190);
border: 4px solid rgb(175, 193, 204);
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string>O</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="ActionLog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>281</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font-weight: normal;
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpUserActions">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123);
color: rgb(255, 255, 255);
font-weight: bold;</string>
    </property>
    <property name="title">
     <string>User Actions</string>
    </property>
    <widget class="QPushButton" name="btnPowerButton_3">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>580</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(175, 193, 204);
font-size: 45px;
font-weight: bold;
background-color: rgb(133, 172, 190);
border: 4px solid rgb(175, 193, 204);
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string>O</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCheckPatient">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font-weight: bold;
border: 4px solid  rgb(175, 193, 204);
border-radius: 6px;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Check Patient</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCallHelp">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font-weight: bold;
border: 4px solid  rgb(175, 193, 204);
border-radius: 6px;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Call Help</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="grpPadOptions">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>301</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 10px</string>
     </property>
     <property name="title">
      <string>Attach</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QPushButton" name="btnAttachChildPads">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 13px;
color: rgb(0, 0, 0);
font-weight: bold;
border: 4px solid  rgb(175, 193, 204);
border-radius: 6px;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Child Pads</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAttachAdultPads">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 13px;
color: rgb(0, 0, 0);
font-weight: bold;
border: 4px solid  rgb(175, 193, 204);
border-radius: 6px;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Adult Pads</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="grpPadOptions_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>301</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 10px</string>
     </property>
     <property name="title">
      <string>Apply</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QPushButton" name="btnApplyCompressions">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 13px;
color: rgb(0, 0, 0);
font-weight: bold;
border: 4px solid  rgb(175, 193, 204);
border-radius: 6px;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Compressions</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnApplyBreathes">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 13px;
color: rgb(0, 0, 0);
font-weight: bold;
border: 4px solid  rgb(175, 193, 204);
border-radius: 6px;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Breaths</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="lblBorder2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>791</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(176, 214, 21);
border: 1px solid rgb(192, 191, 188);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblBorder3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>20</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(176, 214, 21);
border: 1px solid rgb(192, 191, 188);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblBorder4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>680</y>
      <width>791</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(176, 214, 21);
border: 1px solid rgb(192, 191, 188);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblDivider3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>250</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(176, 214, 21);
border: 1px solid rgb(192, 191, 188);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
